controllers/ptpoperatorconfig_controller.go

```diff
@@ -140,6 +145,34 @@ func (r *PtpOperatorConfigReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	return reconcile.Result{RequeueAfter: ResyncPeriod}, nil
 }
 
+// createLeapConfigMap creates an empty leap second config map
+func (r *PtpOperatorConfigReconciler) createLeapConfigMap(ctx context.Context, defaultCfg *ptpv1.PtpOperatorConfig) error {
+	var err error
+
+	cm := &corev1.ConfigMap{}
+	err = r.Get(ctx, types.NamespacedName{
+		Namespace: names.Namespace, Name: names.DefaultLeapConfigMapName}, cm)
+	if err != nil {
+		if errors.IsNotFound(err) {
+			cm.Name = names.DefaultLeapConfigMapName
+			cm.Namespace = names.Namespace
+			cm.Data = make(map[string]string)
+
+			if err = controllerutil.SetControllerReference(defaultCfg, cm, r.Scheme); err != nil {
+				return fmt.Errorf("failed to set owner reference: %v", err)
+			}
+			err = r.Create(ctx, cm)
+			if err != nil {
+				return fmt.Errorf("failed to create leap config map: %v", err)
```

Comment:
just return error since caller already have logged ""failed to create leap config map:"
```
return fmt.Errorf(err)
```